fog/ledger/test_infra/src/lib.rs

```diff
@@ -153,6 +158,10 @@ impl Ledger for MockLedger {
         unimplemented!()
     }
 
+    //    fn add_key_image_data(&self, _key_image: &KeyImage, _data: KeyImageData)
+    // -> Result<Option<u64>, Error> {      unimplemented!()
+    // }
+
     fn get_key_images_by_block(&self, _block_number: u64) -> Result<Vec<KeyImage>, Error> {
```

Comment:
These commented out code should be deleted

Reply:
Done